dll/win32/browseui/internettoolbar.cpp

```diff
@@ -714,45 +710,40 @@ HRESULT CInternetToolbar::CreateMenuBar(IShellMenu **pMenuBar)
 }
 
 HRESULT CInternetToolbar::LockUnlockToolbars(bool locked)
+{
+    if (locked != pSettings->fLocked)
```

Comment:
Can this code be called with `pSettings` still being NULL ? (I hope it shouldn't)

Reply:
It's not possible. `LockUnlockToolbars` is only called in two places. One of those places is the context menu on the control itself. The other is in the view context menu for the parent window.